static/app/components/core/button/index.chonk.tsx

```diff
@@ -159,7 +161,7 @@ export function getChonkButtonStyles(
             },
           },
 
-          '&:disabled': {
+          '&:disabled, &[aria-disabled="true"]': {
```

Comment:
I’ve set `aria-disabled` for the element in the `segmentedControl` that receives styling, so I need to match it here, too.